.github/workflows/test-component-library-vizro-core.yml

```diff
@@ -0,0 +1,66 @@
+name: Integration tests Component Library
```

Comment:
Now we have two sorts of integration tests in this repo the naming is going to get confusing. Let's discuss what to call your tests. Maybe e2e tests?